rbac/management/role/definitions/catalog.json

```diff
@@ -1,179 +1,278 @@
+            {
+              "attribute_filter": {
+                "value": "*",
+                "key": "id",
+                "operation": "equal"
+              }
+            }
+          ]
+        },
+        {
+          "permission": "catalog:progress_messages:write",
+          "resource_definitions": [
+            {
+              "attribute_filter": {
+                "value": "*",
+                "key": "id",
+                "operation": "equal"
+              }
+            }
+          ]
+        },
+        {
+          "permission": "catalog:approval_requests:read",
+          "resource_definitions": [
+            {
+              "attribute_filter": {
+                "value": "*",
+                "key": "id",
+                "operation": "equal"
+              }
+            }
+          ]
+        },
+        {
+          "permission": "catalog:approval_requests:write",
+          "resource_definitions": [
+            {
+              "attribute_filter": {
+                "value": "*",
+                "key": "id",
+                "operation": "equal"
+              }
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "name": "Catalog User",
+      "system": true,
+      "version": 3,
+      "description": "A catalog user roles grants read and order permissions",
+      "access": [
+        {
+          "permission": "catalog:portfolios:read"
+        },
+        {
+          "permission": "catalog:portfolios:order"
+        },
+        {
+          "permission": "catalog:portfolio_items:read"
+        },
+        {
+          "permission": "catalog:portfolio_items:order"
+        },
+        {
+          "permission": "catalog:orders:read",
+          "resource_definitions": [
+            {
+              "attribute_filter": {
+                "value": "{{username}}",
+                "key": "owner",
+                "operation": "equal"
+              }
+            }
+          ]
+        },
+        {
+          "permission": "catalog:orders:order"
+        },
+        {
+          "permission": "catalog:order_items:read",
+          "resource_definitions": [
+            {
+              "attribute_filter": {
+                "value": "{{username}}",
+                "key": "owner",
+                "operation": "equal"
+              }
+            }
+          ]
+        },
+        {
+          "permission": "catalog:order_items:order"
+        },
+        {
+          "permission": "catalog:progress_messages:read",
+          "resource_definitions": [
+            {
+              "attribute_filter": {
+                "value": "{{username}}",
+                "key": "owner",
+                "operation": "equal"
+              }
+            }
+          ]
+        },
+        {
+          "permission": "catalog:approval_requests:read",
+          "resource_definitions": [
+            {
+              "attribute_filter": {
+                "value": "{{username}}",
```

Comment:
@chambridge I hope this doesn't get expanded when you store it in the database

Reply:
Looks fine when run locally. 
```
{
            "uuid": "0540e671-4eb6-4c03-840c-7a3f8f025788",
            "name": "Catalog User",
            "description": "A catalog user roles grants read and order permissions",
            "access": [
                {
                    "permission": "catalog:approval_requests:read",
                    "resourceDefinitions": [
                        {
                            "attributeFilter": {
                                "key": "owner",
                                "value": "{{username}}",
                                "operation": "equal"
                            }
                        }
                    ]
                },
                {
                    "permission": "catalog:portfolios:read",
                    "resourceDefinitions": []
                },
                {
                    "permission": "catalog:portfolios:order",
                    "resourceDefinitions": []
                },
                {
                    "permission": "catalog:portfolio_items:read",
                    "resourceDefinitions": []
                },
                {
                    "permission": "catalog:portfolio_items:order",
                    "resourceDefinitions": []
                },
                {
                    "permission": "catalog:orders:read",
                    "resourceDefinitions": [
                        {
                            "attributeFilter": {
                                "key": "owner",
                                "value": "{{username}}",
                                "operation": "equal"
                            }
                        }
                    ]
                },
                {
                    "permission": "catalog:orders:order",
                    "resourceDefinitions": []
                },
                {
                    "permission": "catalog:order_items:read",
                    "resourceDefinitions": [
                        {
                            "attributeFilter": {
                                "key": "owner",
                                "value": "{{username}}",
                                "operation": "equal"
                            }
                        }
                    ]
                },
                {
                    "permission": "catalog:order_items:order",
                    "resourceDefinitions": []
                },
                {
                    "permission": "catalog:progress_messages:read",
                    "resourceDefinitions": [
                        {
                            "attributeFilter": {
                                "key": "owner",
                                "value": "{{username}}",
                                "operation": "equal"
                            }
                        }
                    ]
                }
            ]
```

<img width="662" alt="image" src="https://user-images.githubusercontent.com/29379759/56970926-30a47100-6b36-11e9-9536-dc821d4ce975.png">